lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -906,35 +903,43 @@ def exec_command_output(self, command):
         Returns:
             output of command execution
         """
-        data = self._data
-        data["command"] = '"' + command + '"'
-        param_list = shlex.split(self._command.format(**data))
+        param_list = shlex.split(self._command.format(**self._credentials, command=f'"{command}"'))
         return subprocess.check_output(
             param_list,
             shell=False,
         ).decode("utf-8")
 
+    def exec_command_capture_output(self, command):
+        """Run a command remotely on magma_dev VM.
+
+        Args:
+            command: command (str) to be executed on remote host
+            e.g. 'sed -i \'s/config1/config2/g\' /etc/magma/mme.yml'
+
+        Returns:
+            output of command execution
```

Comment:
How is this function `exec_command_capture_output` different from `exec_command_output`? As per the description both these functions return the same thing, i.e, command's output

Reply:
I made some more changes that obviate the need for this method.

The difference was that `exec_command_capture_output` captures all output including the stdout/stderr and the return code, and also does not raise an exception if the command returns a non-zero exit code. This could eventually be useful for other applications.

Reply:
I added type hints and additional documentation. The difference is that this one does not raise an exception if the executed command returns a non-zero output.

Reply:
Nice. Thanks for adding the documentation. Now the comment makes the need of this function clear.